src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const robotKeys = Object.keys(robot);
+  const newObj = {};
+
+  for (const value of robotValues) {
+    if (robotValues.indexOf(value) !== robotValues.lastIndexOf(value)) {
+      return null;
+    }
+  }
+
+  for (let i = 0; i < robotKeys.length; i++) {
```

Comment:
Try to rewrite it using only one loop.

1.  check if we already have this key in `newObj`
2.  add key to `newObj`